iocage_lib/ioc_fetch.py

```diff
@@ -505,10 +505,32 @@ def fetch_http_release(self, eol, _list=False):
             silent=self.silent)
         self.fetch_download(self.files)
         missing = self.__fetch_check__(self.files)
+        missing_attempt = 0
 
-        if missing:
-            self.fetch_download(missing, missing=True)
-            self.__fetch_check__(missing, _missing=True)
+        while True:
+            if not self.files_left:
+                break
+
+            if missing_attempt == 4:
+                iocage_lib.ioc_common.logit(
+                    {
+                        'level': 'EXCEPTION',
+                        'message': 'Max retries exceeded, too many failed'
+                                   ' verifications!'
```

Comment:
`too many failed verifications` reads very funny. Verifications of what? The user is clueless. Can we say something alone the lines that the file fetch failed? Verification is a detail of failed fetch.